### ISSUE TITLE
[Design] Layout

### ISSUE THREAD

**Issue by ludiproductions** [lagtv/lagtv#190]
http://www.lag.tv/forums/forums/16/topics/258#post-2702

Um, I figure I get some opinions from you guys. It would have been nice to have this layout before we started on this website, lol. We could work towards this in the future. But for now, no because we worked on this for a while and if we change to this layout/setup now, we're going to possibly have a bad time, lol. There might be alot of break/fix issues @_@ oh boy!
